ParallaxGen/cfg/default.json

```
@@ -12,5 +12,25 @@
     "**/water/**",
     "calientetools/**"
   ],
-  "texture_maps": {}
+  "texture_maps": {},
+  "bsa_exclusionlist": [
```

I'd rather rename this `vanilla_bsas` or something, since otherwise the current name implies everything from the BSA will be excluded
```suggestion
  "vanilla_bsas": [
```

---

You are right, in my mind "exclusion" instead of "excluded" made it clear, but that was nonsense